### DISCUSSION TITLE
Add only subdomain to a project using cli - now domains add

### DISCUSSION THREAD

**Discussion by 1pratham** [vercel/vercel#4330] [General]
Hi All,

I have setup my custom domain with zeit/vercel using the nameserver method and able to manage its DNS records using now cli. The root domain site is hosted someplace else. Now to a project, I want to add a subdomain lets say app.mydomain.com. So, I enter the following command:

`now domains add app.mydomain.com`

However, it gives the below output:
Error! You are adding 'app.mydomain.com' as a domain name containing a subdomain part 'app'
This feature is deprecated, please add just the root domain: `now domain add mydomain.com`

I only want to add a subdomain and not a root domain.

**Comment by AxelZouebi** [comment]
Hello, have you succeeded ?
